qualtricssurvey/public/view.less

```diff
@@ -1,3 +1,5 @@
 .qualtricssurvey_block {
-    // TODO: Put your styles here
+    .qualtrics-button {
+        color: white !important;
```

Comment:
there's a bunch of selectors in platform that sets `a:visited` to a dark color, which looks funky. I guess I can just make my selector `.qualtrics-button:visited`